tools/Crank/benchmarks.yml

```diff
@@ -0,0 +1,36 @@
+imports:
+  - https://raw.githubusercontent.com/dotnet/crank/master/src/Microsoft.Crank.Jobs.Bombardier/bombardier.yml
+
+jobs:
+  server:
+    source:
+      repository: https://github.com/Azure/azure-functions-host
+      branchOrCommit: "{{ FunctionsHostBranchOrCommit }}"
+      project: src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj
```

Comment:
Is there a flag for the run description? In the output I see "Running session '6e9dde64f0654320a942f484b05d3114' with description ''". Might be good to allow a description to be passed through, for when we have reporting. E.g. "Fixing metrics hot lock" for example.

Reply:
There should be a way, I'll figure this out for the subsequent PRs.